stable/xray/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: xray
-version: 2.0.2
-appVersion: 2.12.1
+version: 2.0.3
```

Comment:
Should be 2.1.0 as xray version also has a new minor versiom.

Reply:
+1

Reply:
Corrected now.